pkgs/development/libraries/intel-media-sdk/default.nix

```diff
@@ -18,6 +18,11 @@ stdenv.mkDerivation rec {
   ];
   nativeCheckInputs = [ gtest ];
 
+  # For forward compatibility with oneVPL.
+  # To ensure the VPL backend is dispatched at runtime, export INTEL_MEDIA_RUNTIME=ONEVPL
+  # See https://github.com/Intel-Media-SDK/MediaSDK#media-sdk-support-matrix
+  runtimeInputs = [ oneVPL ];
+
```

Comment:
This means it still works on older GPUs but it uses oneVPL on newer ones, right?

Reply:
yes, it only uses oneVPL at runtime if the environment variable `INTEL_MEDIA_RUNTIME` is `ONEVPL`